### DISCUSSION TITLE
mingw32-make PLATFORM=PLATFORM_DESKTOP won't work

### DISCUSSION THREAD

**Discussion by CreshFreamer** [raysan5/raylib#2411] [Q&A]
I used mingw32-make PLATFORM=PLATFORM_DESKTOP, spams me with errors. Tried mingw32-make clean and nothing found, the final error message from the thousands that I get is "Makefile:561: recipe for target 'core.o' failed
mingw32-make: *** [core.o]"

**Comment by raysan5** [comment]
@CreshFreamer I'm afraid you are not providing enough details for anyone to help... I recommend that you join raylib Discord and ask in help-setup channel.

**Reply by CreshFreamer** [reply]
> @CreshFreamer I'm afraid you are not providing enough details for anyone to help... I recommend that you join raylib Discord and ask in help-setup channel.

I don't have discord, here are some screenshots of the problem 
![image](https://user-images.githubusercontent.com/79119586/160182312-13b5b036-657f-4d98-a072-bab22e637eb2.png)
![image](https://user-images.githubusercontent.com/79119586/160182463-f6fcc13d-fa5b-4372-bef7-f61a34b5e255.png)
I don't have a discord cause reasons don't ask

**Comment by raysan5** [comment]
@CreshFreamer No idea, never seen this issue before.

**Reply by CreshFreamer** [reply]
> @CreshFreamer No idea, never seen this issue before.

Yea, im sad I'll have to use Unity or another game engine since both Raylib and SDL2 haven't worked with me, and I don't know if switching to notepad++ will fix the problem but I don't like notepad++ tho, and btw when I restart my computer I'm able to run the main.c example code for VSCode but only 1 time, because if I modify the file in the slightest it just tells me <raylib.h> no such file or directory.

**Reply by CreshFreamer** [reply]
gonna check in windows sandbox real quick let's see if it's something conflicting or anything I don't even know why any library won't work with me im shaking and crying rn libraries hate me I don't like unity cause it cringe cause the funni haha xd cringe 0 dollar unity watermark=cringe funi

**Reply by CreshFreamer** [reply]
alright got new information, didn't work in windows sandbox, but my actual pc and the windows sandbox thing have something in common: Makefile:576: recipe for target 'rmodels.o' failed

**Comment by CreshFreamer** [comment]
@raysan5 it's something related to the make file: Makefile:561: recipe for target 'core.o' failed
mingw32-make: *** [core.o] Error 1
